topic/src/main/java/tech/ydb/topic/read/impl/ReaderImpl.java

```diff
@@ -259,6 +259,7 @@ private void sendReadRequest() {
             if (currentSizeBytesToRequest <= 0) {
                 logger.debug("[{}] Nothing to request in DataRequest. sizeBytesToRequest == {}", fullId,
                         currentSizeBytesToRequest);
+                sizeBytesToRequest.addAndGet(currentSizeBytesToRequest);
```

Comment:
Negative size is unreal afaik.
Zero is pretty possible in case of race condition between two threads.
Fix is kinda paranoic but OK.
I'd check for zero before modifying Atomic member